gbfs.md

```diff
@@ -76,6 +79,19 @@ system_pricing_plans.json | Optional | System pricing scheme.
 system_alerts.json | Optional | Current system alerts.
 geofencing_zones.json <br/>*(added in v2.1-RC)* | Optional | Geofencing zones and their associated rules and attributes.    
 
+## Accessibility 
+Datasets should be published at an easily accessible, public, permanent URL. (e.g., www.agency.org/gbfs/gbfs.json). Ideally, the URL should be directly available without requiring login to access the file to facilitate download by consuming software applications. While it is recommended (and the most common practice) to make a GBFS dataset openly downloadable, if a data provider does need to control access to GBFS for licensing or other reasons, it is recommended to control access to the GBFS dataset using API keys, which will facilitate automatic downloads.  
+
+To be compliant with GBFS, all systems must have an entry in the [systems.csv](https://github.com/NABSA/gbfs/blob/master/systems.csv) file.
```

Comment:
This does not apply to private feeds, right?

```suggestion
To be compliant with GBFS, all openly available systems must have an entry in the [systems.csv](https://github.com/NABSA/gbfs/blob/master/systems.csv) file.
```

Reply:
Currently to be fully compliant feeds should be public and have an entry in systems.csv. That hasn't been made as clear as maybe it should have been and we're hoping to clear some of it up with this proposal. We're aware of the need for private feeds and we're planning work on private feeds in the near future. We'll be sure to make it clear what full compliance means in the case of private feeds.

Reply:
That makes sense. Thanks for clarifying.